STYLE_GUIDE.md

```diff
@@ -0,0 +1,461 @@
+# Pony Standard Library Style Guide
+
+This is a document describing the preferred practices for writing code for the Pony standard library. Other projects are free to use the same style, but you are free to use any style in other projects. The following conventions are meant to ensure consistency and readability of Pony code. Any of the following rules may be broken in situations where the recommended practice would diminish readability.
+
+
+## Formatting
+
+### Line Length
+
+The maximum line length is 80 columns.
```

Comment:
OK, 80 it is. I've given in and always wear my readers when on the computer, which then lets me use a 10pt font on my retina 15" MacBook Pro to get about 300 columns across the screen. But I guess I'm an outlier.